packages/thirdweb/src/exports/wallets/in-app.ts

```diff
@@ -4,5 +4,7 @@ export {
   preAuthenticate,
   authenticate,
   getUserEmail,
+  getUserPhoneNumber,
+  getAuthenticatedUser,
```

Comment:
waas there a reason to expose `getAuthenticatedUser` ? the response type is a bit messy right now and is likely gonna change, that's why i didnt expose the whole thing.

can we get away with just exposing `getUserEmail` and `getUserPhoneNumber` for now?